package on2015_02.on2015_02_21_AdInfinitum10.FunWith1010;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("lib/test/on2015_02/on2015_02_21_AdInfinitum10/FunWith1010/FunWith1010.task"))
			Assert.fail();
	}
}
